ui/src/app/views/main.html

```diff
@@ -28,7 +28,9 @@ <h1>Bassa</h1>
                 <i class="material-icons">menu</i>
                 </md-button>
                 <md-menu-content width="3">
-                    <md-menu-item><md-button ng-click="logout()" ui-sref="login">Log out from {{ username }} </md-button></md-menu-item>
+                    <md-menu-item><md-button ng-click="" ui-sref="#">Profile</md-button></md-menu-item>
+                    <md-menu-item><md-button ng-click="" ui-sref="#">Settings</md-button></md-menu-item>
+                    <md-menu-item><md-button ng-click="logout()" ui-sref="login">Log out</md-button></md-menu-item>
```

Comment:
This is just some lines added means only UI is changed but you have to make them work like adding some js code for their functionality 😄 will be fine.

Reply:
True

Reply:
@vivonk i think this pr should be closed.